### DISCUSSION TITLE
make Python API (make: *** No rule to make target 'PythonAPI'.  Stop.

### DISCUSSION THREAD

**Discussion by jliu1118** [carla-simulator/carla#5108] [Q&A]
I currently have Carla 0.9.11 installed running on Ubuntu 20.04.3; however every time I'm running the command make Python API I'm getting No rule to make target 'Python API'. I'm really new to both linux and carla. Any help would be appreciated! 

Would downloading a compiled version fix this problem?

**Comment by Raimund69** [comment]
The same for me: I followed https://carla.readthedocs.io/en/0.9.14/build_windows/ 
Everything went fine, until section "1. Compile the Python API client", when the command <make PythonAPI> shows the same error.
Any suggestions?
Thank you in advance!

**Reply by Raimund69** [reply]
solution: for me it was sufficient to restart my computer, now everything's ok.

**Reply by enter802** [reply]
the restart did not work for me. any other tips?

**Comment by dangli1989** [comment]
cd into the path which contains PythonAPI. You are probably in the wrong directory
